src/java.desktop/share/native/libfontmanager/freetypeScaler.c

```diff
@@ -44,6 +44,16 @@
 #include FT_LCD_FILTER_H
 #include FT_MODULE_H
 
+// Linux is built with system Freetype by default,
+// and it's often a bit old and doesn't have FT_COLOR_H.
+// Thus, we disable colored outlines on Linux to be able
+// to build on older Linuxes, this is not a big problem,
+// as Linux uses bitmap emoji anyway.
+#if defined(_WIN32) || defined(__APPLE__)
```

Comment:
Do we really have to disable coloured outlines because of what's available at build time ? Why can't we dynamically locate all the needed symbols at runtime ?

Reply:
Fixed